io.github.atari800.Atari800.yml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.atari800.Atari800
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: atari800
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --socket=pulseaudio
+  - --persist=.
+  - --filesystem=xdg-documents
+
+cleanup:
+  - '/bin/sdl-config'
+  - '/include'
+  - '/lib/pkgconfig'
+  - '*.a'
+  - '*.la'
+
+modules:
+  - name: libsdl
+    sources:
+      - type: archive
+        url: https://libsdl.org/release/SDL-1.2.15.tar.gz
```

Comment:
I think you should use the shared-modules to build this:

https://github.com/flathub/shared-modules/blob/master/SDL/SDL-1.2.15.json

The top-level README explain how to use it.

Reply:
I don't have access to git to commit a change to add the submodule.

Reply:
Can I limit this to x86_64 only?

Reply:
why?

Reply:
It's failing on the arm64 build, right?

Reply:
you refused to do what I recommended.

Reply:
I can't do that because I don't have access to the git repo.  You are deliberately trying to fail this.  And, you know what?  You win.  I give up.  I have no more time to waste on this.  I got it working on my Steam Deck and that's all I wanted.  F everybody else.